test/connections.js

```diff
@@ -316,9 +316,12 @@ test('server responds and immediately ends, multiple connects', async function (
   for (let i = n; i > 0; i--) {
     const socket = b.connect(server.publicKey)
 
-    socket.on('open', () => {
-      if (--n === 0) lc.pass()
-    })
```

Comment:
The socket was throwing an `ECONNRESET` error after moving to UDX. This is perfectly aligned with TCP behaviour though, so that seems intended.